packages/core-data/src/reducer.js

```diff
@@ -234,6 +234,45 @@ export function hasUploadPermissions( state = true, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the most recent autosave.
+ *
+ * @param  {Object} state  Current state.
+ * @param  {Object} action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function autosave( state = {}, action ) {
+	switch ( action.type ) {
+		case 'RESET_AUTOSAVE':
+			const { postId, autosave: autosavePost } = action;
+			const [ title, excerpt, content ] = [
+				'title',
+				'excerpt',
+				'content',
+			].map( ( field ) => {
```

Comment:
To me, given that the intent of the `core-data` package is to serve as providing the raw underlying data, I don't know that we should be doing any post-processing here. I'd expect an autosave "entity" to be exactly what is provided through the REST API.

Reply:
Seems as though there are three main consumers of the data:
- The deprecated selectors.
- The `requestPostUpdate` effect uses `getAutosave`, which would now return a different data structure.
- The `isEditedPostAutosaveable` selector also compares the autosave, passed in from `getAutosave`.

In those places I could use a function to transform the data, `getSavedAutosaveAttributes( autosave )`. I'm not sure where that function should live. Any thoughts?

Reply:
Would it be enough to just use the `getPostRawValue` function from the editor store's `reducer.js`?